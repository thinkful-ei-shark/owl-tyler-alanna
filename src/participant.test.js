import React from 'react';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import Participant from './participant';


//Smoke Test
it ('renders without crashing', ()=> {
    const div = document.createElement('div');

    ReactDOM.render(<Participant name='Jane Doe' avatar='https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1' onStage = {true} inSession={true}/>, div);

    ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Participant name='Jane Doe' avatar='https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1' onStage = {true} inSession={true}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });