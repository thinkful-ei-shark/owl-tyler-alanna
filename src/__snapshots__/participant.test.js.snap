// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="participant"
>
  <img
    alt=""
    src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
  />
  <h3>
    Jane Doe
  </h3>
  <p
    className="in-session"
  >
    On stage
  </p>
  ;
</div>
`;
